easybuild/easyconfigs/o/OpenBLAS/OpenBLAS-0.3.7-GCC-8.3.0-2.32.eb

```diff
@@ -0,0 +1,28 @@
+name = 'OpenBLAS'
+version = '0.3.7'
+
+homepage = 'http://xianyi.github.com/OpenBLAS/'
+description = "OpenBLAS is an optimized BLAS library based on GotoBLAS2 1.13 BSD version."
+
+toolchain = {'name': 'GCC', 'version': '8.3.0-2.32'}
```

Comment:
@bartoldeman This should be changed to just use `8.3.0` as version, since we're dropping the `binutils` version going forward.

You'll need the `GCC` easyconfig from #8567 though of course for that...

Reply:
There's a circular dependency here. Is #8567 merged first (because of `GCC/8.3.0`) or this one?
Or simply create a separate PR for `GCC/8.3.0` + deps, and breaking that off from #8567?

Reply:
@bartoldeman The easiest way forward would be updating #8567 to use OpenBLAS 0.3.7, I guess...